saskatoon/harvest/signals.py

```diff
@@ -38,6 +38,32 @@ def changed_by(sender, instance, **kwargs):
     else:
         instance.changed_by = None
 
+def notify_unselected_pickers(sender, instance, **kwargs):
+    if instance.id:
+        original_instance = sender.objects.get(id=instance.id)
+        if instance.status == "Ready" and original_instance.status == "Date-scheduled":
+            unselected_pickers = instance.get_unselected_pickers()
+            email_list = list()
+            mail_subject = _("[Saskatoon] Not Selected as a picker for upcoming Harvest")
+            for p in unselected_pickers:
+                picker_name = p.picker.name()
+                picker_email = list()
+                picker_email.append(p.picker.email())
+                harvest_details = instance.__str__()
+                message = (_("Hi") + " " + picker_name + ",\n\n" +
+                    _("We are sorry but enough pickers have already been " +
+                    "selected for the ") + harvest_details + ". " +
+                    _("You may still be contacted by the pick-leader " +
+                    "if some of them end up cancelling. We will do our best to " +
+                    "prioritize your participation next time you submit a request " +
+                    "re-using the same email for another harvest.") + "\n\n" +
+                    _("Thanks for supporting your community!") + "\n\n" +
+                    _("Yours") + ",\n" +
+                    "--\n" +
+                    "Saskatoon Harvest System")
```

Comment:
I'm wondering if using a single string with the `format` method will not make translations easier. What do you think of trying with a syntax like this instead?
```
_("Hi  {} \n\n \
We are sorry but enough pickers have already been selected for the {}. \
You may still be contacted by the pick-leader if some of them end up cancelling. \ 
We will do our best to prioritize your participation next time you submit a request \
with the same email address. \n\n \
Thanks for supporting your community! \n\n \
Yours, \n -- \n \
Saskatoon Harvest System").format(picker_name, harvest_details)
```

Reply:
I tested and its working fine. Have updated the PR.